modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/data/commerce_theme_portlet_settings.json

```
@@ -24,7 +24,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
+				"FileName": "minium_sub_site_navigation.ftl",
```

Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
```